app/assets/stylesheets/spree/frontend/base/_fonts.scss

```diff
@@ -1,52 +1,52 @@
-$inter-font-path: 'base/fonts/inter/';
+$inter-font-path: 'spree/frontend/base/fonts/inter/';
 
 @font-face {
   font-family: 'Inter';
   font-style: normal;
   font-weight: 300;
   font-display: swap;
-  src: url('#{$inter-font-path}Inter-Light.woff2?v=3.15') format('woff2'),
-       url('#{$inter-font-path}Inter-Light.woff?v=3.15') format('woff');
+  src: font-url('#{$inter-font-path}Inter-Light.woff2?v=3.15') format('woff2'),
```

Comment:
I based this update from https://github.com/solidusio/solidus/blob/5cd7e9c5ebff201102030ddc4d5d60ef62d65eee/backend/app/assets/stylesheets/spree/backend/shared/_fonts.scss.